### PR TITLE
Feature/(cu bfth9g) change test format from mocha chai to jest

### PR DESCRIPTION
**Pull request by Lucria**

### CODE REVIEW

**Comment by AtomiCloudAdmin** [comment]
Task linked: [CU-bfth9g Change test format from mocha chai to jest](https://app.clickup.com/t/bfth9g)

File: jest.config.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    coverageThreshold: {
+        global: {
+            "branches": 95,
+            "functions": 95,
+            "lines": 95,
+            "statements": -10
+        }
+    },
+    transformIgnorePatterns: [
```
Comment (kirinnee) [inline jest.config.ts]: Remove this

File: jest.config.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+// Sync object
+const config: Config.InitialOptions = {
+    preset: "ts-jest",
+    testEnvironment: "node",
+    verbose: true,
+    testMatch: ["**/__tests__/**/*.ts?(x)", "**/?(*.)+(spec|test).ts?(x)"],
+    coverageDirectory: "./coverage",
+    coverageReporters: ["json", "lcov", "text", "clover"],
+    transform: {
```
Comment (kirinnee) [inline jest.config.ts]: remove this too

**Review by kirinnee — APPROVED** [review]
LGTM 😎

**Pull request merged**